test/parallel/test-internal-modules.js

```diff
@@ -1,13 +1,31 @@
+// Flags: --expose-internals
 'use strict';
-const common = require('../common');
-const path = require('path');
+require('../common');
+
 const assert = require('assert');
+const internalModule = require('internal/module');
+
+assert.strictEqual(
+  internalModule.stripShebang(''),
+  ''
+);
+
+assert.strictEqual(
+  internalModule.stripShebang('aa'),
+  'aa'
+);
+
+assert.strictEqual(
+  internalModule.stripShebang('#!'),
+  ''
+);
 
-assert.throws(function() {
-  require('internal/freelist');
-}, /^Error: Cannot find module 'internal\/freelist'$/);
+assert.strictEqual(
+  internalModule.stripShebang('#!bin/bash'),
+  ''
+);
 
 assert.strictEqual(
-  require(path.join(common.fixturesDir, 'internal-modules')),
-  42
+  internalModule.stripShebang('#!bin/bash\naa'),
+  '\naa'
```

Comment:
Something like the following would be far more compact (and likely easier to maintain):

```js
[
  ['', ''],
  ['aa', 'aa'],
  ['#!', ''],
  ['#!bin/bash', ''],
  ['#!bin/bash\naa', '\naa']
].forEach((i) => assert.strictEqual(internalModule.stripShebang(i[0]), i[1]));
```